octavia_f5/controller/worker/sync_manager.py

```diff
@@ -189,15 +234,18 @@ def tenant_delete(self, network_id):
         :param network_id: network id
         :return: requests delete result
         """
-        tenant = m_part.get_name(network_id)
+        if CONF.f5_agent.dry_run:
+            return FakeOK
```

Comment:
I think you meant to instantiate the class `FakeOK` (like it's done in the `MonitorDeletionException` catch below) instead of returning the class itself.